test/xds/xds_client_affinity_test.go

```diff
@@ -1,6 +1,3 @@
-//go:build !386
```

Comment:
Why was this here before?

Reply:
I can't remember exactly. But the one issue I remember is with go-control-plane, where they had an int64 in a struct which was accessed atomically, but wasn't aligned on 64-bit boundaries. I fixed it, but we had to wait a while for a release. So, I think we started adding these and over time, I'm guessing newer files just mimicked the existing ones.